PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetEnergyFlow.h

```diff
@@ -66,6 +67,7 @@ class AliAnalysisTaskEmcalJetEnergyFlow: public AliAnalysisTaskEmcalJet {
 	void                    DoClusterLoop()                         ; ///<May remove later	
 	void                    DoCellLoop()                            ; ///<May remove later
 
+        Bool_t                  isMCprod                                ;/// Flag for MC productions
```

Comment:
If you're streaming this (it's ambiguous from your comment, but based on the header, I think you want to stream it), then you need to bump the classdef